apps/snmp_exporter/app.yaml

```diff
@@ -2,7 +2,7 @@
     "id": "/snmp_exporter",
     "cmd": null,
     "cpus": 1,
-    "mem": 1024,
+    "mem": 65,
```

Comment:
Strange size, but ok.....

Reply:
lol i guess I can make it 64 so it's a power of 2